test/Microsoft.AspNet.Mvc.FunctionalTests/JsonResultTest.cs

```diff
@@ -161,6 +161,30 @@ public class JsonResultTest
             Assert.Equal("{\"message\":\"hello\"}", content);
         }
 
+        [Theory]
+        [InlineData("application/json")]
+        [InlineData("text/json")]
+        public async Task JsonResult_CustomSerializerSettings_Conneg(string mediaType)
```

Comment:
Suggest make it a Fact and a simple name like 
`JsonResult_Uses_CustomSerializerSettings`